src/main/java/org/elasticsearch/script/CompiledScript.java

```diff
@@ -19,22 +19,23 @@
 
 package org.elasticsearch.script;
 
+import static org.elasticsearch.script.ScriptService.ScriptType;
+
 /**
  *
  */
```

Comment:
Would like to note this was how it was when I began working on this, but I'm happy to make the fix.